.env.docker.example

```
@@ -0,0 +1,28 @@
+# AI SERVICE
+
+AI_SERVICE_PORT=8000
+EMBEDDING_MODEL=sentence-transformers/all-MiniLM-L6-v2
+CHROMA_STORE_PATH=./chroma_store
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHROMA_STORE_PATH key should go before the EMBEDDING_MODEL key

(UnorderedKey)